<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dailymsy_allin</class>
 <widget class="QDialog" name="dailymsy_allin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>432</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>432</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>432</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>265</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文琥珀</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>任务描述：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="thingsname">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文琥珀</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>该任务有多重要：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文琥珀</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>该任务有多紧急：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="missionAdd">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>300</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;华文琥珀&quot;;</string>
   </property>
   <property name="text">
    <string>添加任务</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="imNum">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>300</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="emNum">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>330</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="workTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>461</width>
     <height>251</height>
    </rect>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>序号（id）</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>事件描述（ThingNames）</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>重要程度（im）</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>紧急程度（em）</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="showMission">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>369</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文琥珀</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>当前选中：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteMission">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>380</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;华文琥珀&quot;;</string>
   </property>
   <property name="text">
    <string>我完成这项任务辣</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delectAll">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>360</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;华文琥珀&quot;;</string>
   </property>
   <property name="text">
    <string>和所有烦恼说拜拜</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextRow">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>380</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;华文琥珀&quot;;</string>
   </property>
   <property name="text">
    <string>转到下一行</string>
   </property>
  </widget>
  <widget class="QLabel" name="show">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>380</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>304</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>330</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="pic">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>201</width>
     <height>251</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
